spec/lib/orbf/rules_engine/printers/graphviz_project_printer_spec.rb

```diff
@@ -40,4 +40,30 @@
   it "simple project" do
     expect(subject.print_project(project).join("\n")).to eq(fixture_content(:rules_engine, :printers, "graphviz_simple.txt"))
   end
+
+  describe "#print_packages" do
+    rule_kinds = Orbf::RulesEngine::Rule::Kinds.all - [Orbf::RulesEngine::Rule::Kinds::PAYMENT]
+    rules = rule_kinds.collect do |kind|
+
+      it "can handle rule of kind #{kind}" do
+        rule = Orbf::RulesEngine::Rule.new(
+          kind:     kind,
+          formulas: [Orbf::RulesEngine::Formula.new(
+                       "form_for_#{kind}",
+                       "quantity_act1_verified_for_1_and_201601 * 33",
+                       "comment_for_#{kind}"
+                     )]
+        )
+        package = Orbf::RulesEngine::Package.new(
+          code:       :facility,
+          kind:       :single,
```

Comment:
I think we are missing an invariant enforcement in the Package in the after_init method

```ruby
zone_related_rules = package.rules.select(&:zone_related?)
raise "#{zone_related_rules.map(&:code)} rules are zone related but the package isn't zone related"  if zone_related_rules.any? && !package.zone_kind? 
```